### PR TITLE
SplunkPy - parse-raw command enhancement 

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/12004
https://github.com/demisto/etc/issues/19802

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer
image

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests

### CODE REVIEW

**Comment by kirbles19** [comment]
@Shellyber Done.

File: Integrations/SplunkPy/SplunkPy.py [hunk 1]
```diff
@@ -67,52 +67,20 @@ def get_current_splunk_time(splunk_service):
...
-            key_val = key_val.strip()
-            if '=' in key_val:
-                key_and_val = key_val.split('=')
-                parsed_raw[key_and_val[0]] = key_and_val[1]
-        result.update(parsed_raw)
-
-        return parsed_raw
+    raw_ = re.split('\S,', raw)
+    for key_val in raw_:
+        key_val = key_val.replace('"', '')
```
Comment (Itay4) [inline Integrations/SplunkPy/SplunkPy.py]: do both `replace` and `strip` in the same line, and don't assign and initialize same arg, use different arg name

File: Integrations/SplunkPy/SplunkPy_test.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import unittest
```
Comment (Itay4) [inline Integrations/SplunkPy/SplunkPy_test.py]: use `pytest` instead

File: Integrations/SplunkPy/SplunkPy_test.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    "info_min_time": '0.000',
+    "search_name": "NG_SIEM_UC25- High number of hits against unknown website from same subnet",
+    "src": "xx.xx.xxx.xx , yyy.yy.yyy.yy , zz.zzz.zz.zzz , aa.aa.aaa.aaa",
+    "u_category": "unknown",
+    "user": "xyz\\a1234 , xyz\\b5678 , xyz\\c91011 , xyz\\d121314 , unknown",
+    "website": "2.2.2.2"
+}
+
+
+class TestRawToDict(unittest.TestCase):
```
Comment (Itay4) [inline Integrations/SplunkPy/SplunkPy_test.py]: no need from class here

File: Integrations/SplunkPy/SplunkPy_test.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    def test_raw_to_dict(self):
+        from SplunkPy import rawToDict
+        actual_raw = DICT_RAW_RESPONSE.replace('\n', '')
```
Comment (Itay4) [inline Integrations/SplunkPy/SplunkPy_test.py]: better to initialize `DICT_RAW_RESPONSE` without `\n` in the first place

File: Integrations/SplunkPy/SplunkPy.py [hunk 5]
```diff
@@ -68,34 +69,18 @@ def get_current_splunk_time(splunk_service):
...
-
-            alreadyThere = False
-            for dictkey, dictvalue in result.items():
-                if dictkey == key:
-                    alreadyThere = True
-                    result[dictkey] = dictvalue + "," + val
-
-            if not alreadyThere:
-                result[key] = val
+    raw_ = re.split('\S,', raw)
```
Comment (Itay4) [inline Integrations/SplunkPy/SplunkPy.py]: lets improve argument name here and add a comment what are we doing here

**Comment by Shellyber** [comment]
@Itay4 Please review

**Pull request merged**